### DISCUSSION TITLE
support http scale with multiple services/deployments seperatly

### DISCUSSION THREAD

**Discussion by HongboFei** [kedacore/http-add-on#540] [Q&A]
Scenarios: We have 3 or more http service/deployment (frontend, backend and algorithm).  We want to only scale up algorithm service/deployment to more instances/pods ONLY, frontend and backend should not scale up as many as the algorithm. We don't have much traffic happens in front and backend. Only the algorithm needs to scale up. 
Looks like current https add-ons cannot support that. Once the scale up is triggered, i see all the deployments (frontend, backend and algorithm) are scaled up. 
Thanks.

**Comment by HongboFei** [comment]
Looks like can be achieved by https://github.com/kedacore/http-add-on/issues/338, will try though.

**Comment by tomkerkhove** [comment]
It comes back to how you compose your apps - Do you have separate deployments for them? What is preventing you to achieve this with what you have today?

**Reply by HongboFei** [reply]
Basically, same problem with #338. Only one host, different subpath map to different deployment/service. 
e.g. Host is example.com, example.com/api1 will map to deployment api1, example.com/api2 will map to deployment api2, and may have api 3,4..... 
Expect is when call example.com/api1, should omly scale on deployment api1. Seems not support that yet.
